OADP-4003: Update OADP and Velero attributes

apurvabhide17 opened the pull request:
Version(s):
OADP 1.4.0
OCP 4.12 → OCP 4.16

Issue:
[OADP-4003](https://issues.redhat.com/browse/OADP-4003)

Link to docs preview:
https://76841--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator

QE review:
- [x] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

apurvabhide17 commented:
/label OADP

apurvabhide17 commented:
/label peer-review-needed

tmalove commented:
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

weshayutin commented on modules/velero-oadp-version-relationship.adoc:
```diff
@@ -17,5 +17,5 @@
 | 1.2.2 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.2.3 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
-| 1.3.0 | link:https://{velero-domain}/docs/v1.12/[1.12] | 4.12 and later
+| 1.4.0 | link:https://{velero-domain}/docs/v1.14/[1.14] | 4.12 and later
```
OADP 1.4.0 will support OCP 4.14 and later ( update 4.12/4.14 )

weshayutin replied:
Please also do not overwrite 1.3.0, we're also missing 1.3.1

weshayutin reviewed (changes requested):
/hold

weshayutin commented on modules/velero-oadp-version-relationship.adoc:
```diff
@@ -18,4 +18,6 @@
 | 1.2.2 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.2.3 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
```
remove "and later" replace with "OCP 4.10 - 4.15"

apurvabhide17 replied:
thank you, I've updated that.

rayfordj commented on modules/velero-oadp-version-relationship.adoc:
```diff
@@ -18,4 +18,6 @@
 | 1.2.2 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.2.3 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.3.0 | link:https://{velero-domain}/docs/v1.12/[1.12] | 4.12 and later
+| 1.3.1 | link:https://{velero-domain}/docs/v1.12/[1.12] | 4.12 and later
```
```suggestion
| 1.3.0 | link:https://{velero-domain}/docs/v1.12/[1.12] | 4.12 and later
| 1.3.1 | link:https://{velero-domain}/docs/v1.12/[1.12] | 4.12 and later
```

The OADP 1.3 release stream should be v4.10-v4.15

apurvabhide17 replied:
thank you, I've updated that

amastbau commented:
Just apply the latest comments for the version rage and it can be merged! LGTM*  *with a note

apurvabhide17 commented:
> Just apply the latest comments for the version rage and i think it can be merged! LGTM* *with a note

Hello @amastbau, 
I added the required changes. Let me know if it looks good, accordingly I'll squash the commits and it can be merged during the release :) 
Thanks!

amastbau commented:
LGTM @apurvabhide17

weshayutin reviewed (approved):
/LGTM

weshayutin commented:
ignore that comment, cut and paste fail

apurvabhide17 commented:
Thank you, @amastbau, @weshayutin, and @rayfordj, for reviewing it

apurvabhide17 commented:
/label merge-review-needed

adellape commented on _attributes/common-attributes.adoc:
```diff
@@ -58,7 +58,7 @@ endif::[]
 // Backup and restore
 :velero-domain: velero.io
-:velero-version: 1.12
+:velero-version: 1.14.0
```
```suggestion
:velero-version: 1.14
```
Setting this to `1.14.0` creates a URL that 404s in the intro of https://76841--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.

From https://github.com/openshift/openshift-docs/blob/main/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc?plain=1#L11

```
The OADP Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
```

apurvabhide17 replied:
@adellape, thanks for sharing this. I've fixed it

adellape reviewed (commented):
Holding merge so as not to create a URL that 404s. Please re-queue for merge-review when ready, or ping me on Slack. Thank you!

adellape commented:
Also, the PR description mentions OCP 4.12 - 4.16, but I assume it should now also include 4.17. I've added that branch label to the PR for cherrypicking as well, but please correct me if I'm wrong.

apurvabhide17 commented:
> Holding merge so as not to create a URL that 404s. Please re-queue for merge-review when ready, or ping me on Slack. Thank you!

Hello @adellape, I've fixed that error and added the merge-review-needed label again. Could you please merge it now?
Thanks!

adellape commented:
/cherrypick enterprise-4.17

adellape commented:
/cherrypick enterprise-4.16

adellape commented:
/cherrypick enterprise-4.15

adellape commented:
/cherrypick enterprise-4.14

adellape commented:
/cherrypick enterprise-4.13

adellape commented:
/cherrypick enterprise-4.12

apurvabhide17 commented:
Hello @weshayutin, @amastbau, @rayfordj,
The merge conflicts for 4.13 and 4.12 are now fixed. Changes for 4.13 are now merged in this [PR](https://github.com/openshift/openshift-docs/pull/78720).
Could you confirm if it is ok to push these changes to 4.12? Sharing the [PR](https://github.com/openshift/openshift-docs/pull/78723).
Thanks!
